src/components/data_comps/desp/desp_comp_mod.F90

```diff
@@ -219,8 +219,13 @@ subroutine desp_comp_init(EClock, espid, mpicom_in, phase, read_restart,    &
       ! Initialize PIO
       !------------------------------------------------------------------------
 
-      iosystem => shr_pio_getiosys(trim(inst_name))
-      call shr_strdata_pioinit(SDESP, iosystem, shr_pio_getiotype(trim(inst_name)))
+      call shr_strdata_pioinit(SDESP, COMPID)
+
+      call mpi_comm_rank(global_comm, global_mype, ierr)
+      call mpi_comm_size(global_comm, global_numpes, ierr)
+
+      write(logunit,*)'DESP: I am global rank ',global_mype,' of ',global_numpes
```

Comment:
Is this write statement still necessary / useful?

Reply:
Yes I think that it is and this code is only used in tests.